### ISSUE TITLE
IBEX GUI Block Configuration: Resize Configure Block as "rate/seconds" Setting Does Not Fit. 

### ISSUE THREAD

**Issue by JackEAllen** [ISISComputingGroup/IBEX#7267]
## Where?

- `ibex_gui\base\uk.ac.stfc.isis.ibex.ui.configserver\src\uk\ac\stfc\isis\ibex\ui\configserver\editing\blocks`

- `EditBlockDialog.createDialogArea()`

## How?
When performing manual release testing on the GUI, I discovered that the "rate/seconds" option in the Configure Block window when configuring a new block or editing an existing block did not correctly fit in the window as was covered by the "Mode" dropdown setting. 

![image](https://user-images.githubusercontent.com/32413847/182108416-b3454705-9db9-4fa3-8ff5-4a564c5b59a3.png)

## Reproducible?
* Yes

### To Reproduce
- Open up version 11.0.0 pf the GUI
- Select "Edit Current Configuration" from the configuration tab on the menu bar.
- Navigate to the blocks tab in the "Edit the current configuration" window
- Edit an existing block or add a new block

## Acceptance criteria 
- [ ] Configure Block window default size has increased
- [ ] "rate/seconds" option is now fully visible and not partially hidden by "Mode" dropdown.
- [ ] A squish test exists to check this in the future.

## How to Test 
(**Add before making a PR**)_
- [ ] Follow "To Reproduce" instructions and ensure "rate/seconds" option is now fully visible
- [ ] Run squish test and ensure it passes if implemented as part of this issue

**Comment by Tom-Willemsen** [comment]
Can I just check, are you testing at 1920x1080 resolution (may need to restart IBEX after a resolution change)?

Many items are known not to fit at other resolutions.

**Comment by rerpha** [comment]
Looks fine to me at 1920x1080 at 1x scaling - I think you might have different settings.

**Comment by JackEAllen** [comment]
> Can I just check, are you testing at 1920x1080 resolution (may need to restart IBEX after a resolution change)?
> 
> Many items are known not to fit at other resolutions.

Originally I was not, but having changed my resolution since on all monitors and setting 1x scaling, disabling windows advance scaling, deleting my `\workspace\` dir and restarting IBEX I still seem to be experiencing the issue. As it seems that the issue is only being experienced by me so far, I will mark this as won't fix for the moment and update this issue on what I did to fix this on my system. 
I will probably start by just removing my instance of the client and redeploying a fresh one to my machine

**Comment by rerpha** [comment]
log in and out then try the client, just in case it's a windows issue

**Comment by JackEAllen** [comment]
> log in and out then try the client, just in case it's a windows issue

A log in and out and also a full system restart did not resolve the issue. I have also reverted to my original resolution and have found that aspects of the GUI became less visible than previously at the same resolution on switching two and from two different resolutions:  1920 by 1200 -> 1920 by 1080 -> 1920 by 1200 -> 1920 by 1080. 
Interestingly rebuilding at the expected resolution of 1920 by 1080 only seemed to fix this for me locally on my machine. When building at 1920 by 1200 I experienced the same issue. As this is a one time only thing so far, I think I would like to try and replicate this a few time first with building the client using maven to rule out any build discrepancies that may have occurred.
